File: auto-docs/executables/matlab/Electric_heatmap.m
signin('TestBot', 'r1neazxo9w')

size = 50;
z = zeros(size, size);
for r = 1:size
    for c = 1:size
        z(r,c) = r+c
    end
end

data = {...
  struct(...
    'z', z, ...
    'scl', 'Electric', ...
    'type', 'heatmap')...
};
layout = struct('title', 'Electric');

response = plotly(data, struct('layout', layout, 'filename', 'Electric-heatmap', 'fileopt', 'overwrite', 'auto_open', 'false'));
plot_url = response.url

File: auto-docs/executables/julia/Electric_heatmap.jl
using Plotly

using Plotly
Plotly.signin("TestBot", "r1neazxo9w")

size = 50
z = rand(size, size)
for r = 1:size
  for c = 1:size
        z(r,c) = r+c

data = [
  [
    "z" => z, 
    "scl" => "Electric", 
    "type" => "heatmap"
  ]
]
layout = ["title" => "Electric"]

response = Plotly.plot([data], ["layout" => layout, "filename" => "Electric-heatmap", "fileopt" => "overwrite", "auto_open" => "false"])
plot_url = response["url"]